lib/streamlit/runtime/media_file_manager.py

```diff
@@ -15,35 +15,17 @@
 """Provides global MediaFileManager object as `media_file_manager`."""
 
 import collections
-import hashlib
-import mimetypes
 import threading
-from enum import Enum
-from typing import Dict, Set, Optional, List, Union
+from typing import Dict, Set, Optional, Union
 
-from streamlit import util
 from streamlit.logger import get_logger
-from .stats import CacheStatsProvider, CacheStat
+from .media_file_storage import MediaFileStorage, MediaFileKind
 
 LOGGER = get_logger(__name__)
 
-STATIC_MEDIA_ENDPOINT = "/media"
-PREFERRED_MIMETYPE_EXTENSION_MAP = {
-    "image/jpeg": ".jpeg",
-    "audio/wav": ".wav",
-}
-
-
-class MediaFileType(Enum):
-    # used for images and videos in st.image() and st.video()
-    MEDIA = "media"
-
-    # used for st.download_button files
-    DOWNLOADABLE = "downloadable"
-
```

Comment:
Much of the removed behavior in this file now lives in `MemoryMediaFileStorage`.